.github/workflows/deploy.yml

```diff
@@ -39,4 +39,20 @@ jobs:
         with:
           path: ./dist/*
       - name: Deploy to Pypi
-        uses: pypa/gh-action-pypi-publish@release/v1
\ No newline at end of file
+        uses: pypa/gh-action-pypi-publish@release/v1
+
+  publish-tutorials:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.release.name, 'rc')"
+    steps:
+    - uses: actions/setup-python@v2
+        with:
+          python-version: 3.11
+    - name: Publish tutorials to learning.quantum-computing.ibm.com
+      env:
+        LEARNING_API_TOKEN: ${{ secrets.LEARNING_API_TOKEN_PRODUCTION }}
+        LEARNING_API_ENVIRONMENT: production
+      run: |
+        pip install git+https://github.com/frankharkins/learning-content-tools.git@12a71a904b0283b2ee44b63cd04ab8fe96e0df6f#subdirectory=iql-lesson-sync
+        cd docs/tutorials
```

Comment:
Should this repo be in the qiskit org? Seems a bit odd to have to install something from your personal repo

Reply:
That's true, I'll look into it. Is it a requirement for this PR?

Reply:
Could the tools live in qiskit/documentation? I agree this is an issue for sustainability - it's high bus factor.